docs/user-guide/reading_writing_data.rst

```diff
@@ -573,12 +573,12 @@ From this you can validate and distribute your CDF file.
 Validating a CDF File
 =====================
 
-The :py:class:`~hermes_core.timedata.HermesData` uses the `~spacepy.pycdf.istp` module for CDF validation, in addition to custom
-tests for additional metadata. A CDF file can be validated using the :py:func:`~hermes_core.util.validation.validate` method
-and by passing, as a parameter, the full :py:class:`~pathlib.Path` to the CDF file to be validated::
+HERMES supports validation of CDF files to ensure that they are compliant with the ISTP standards.
+The :py:class:`~hermes_core.timedata.HermesData` delegates to the `~swxsoc.util.validation` module for CDF validation.
```

Comment:
Same here, I think you will need to link to the docs page for `swxsoc.util.validation`

Reply:
^ It looks like this one does link to the `swxsoc` docs. 
https://hermes-core--122.org.readthedocs.build/en/122/user-guide/reading_writing_data.html#validating-a-cdf-file